Documentation/getting-started.md

```diff
@@ -1,11 +1,25 @@
-# Getting Started with Analytics SDK
+# Adobe Analytics Android Extension
 
-## Before starting
+The [`Analytics`](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/) extension represents the Analytics Adobe Experience Platform SDK that is required for sending mobile interaction data from your app to Adobe Analytics.
```

Comment:
The ~~[`Analytics`](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/) represents the Analytics~~ [Adobe Experience Platform ~~SDK~~ - Analytics Extension](https://developer.adobe.com/client-sdks/documentation/adobe-analytics) is required for sending ~~mobile~~ interaction data from the mobile application to Adobe Analytics.